### ISSUE TITLE
There are seperate calibration files for the MPPT Serial Nos

### ISSUE THREAD

**Issue by ac2cz** [ac2cz/FoxTelem#236]
Per Bryce:

ADC Voltage to RTD Resistance (Ohms)
Ohms = -122.816*Vadc+252.0553

Then use the normal RTD temperature conversion (cubic function) for resistance to degrees.

Calibration Files:
SN 40: http://amsatfox.org/repos/Fox1/PSU/MPPT/EngineringDocs/Flight_Boards/MPPT_SN40_Calibration.pdf
SN 41: http://amsatfox.org/repos/Fox1/PSU/MPPT/EngineringDocs/Flight_Boards/MPPT_SN41_Calibration.pdf
SN 5: http://amsatfox.org/repos/Fox1/PSU/MPPT/EngineringDocs/Flight_Boards/MPPT_Rev2-1_SN5/MPPT_Rev%202.1%20_SN5_Testing.pdf

Please note that each panel MPPT has a measured Vref. This will generally be somewhere around 2.5V or so but the TL1451 is from the 1970’s so it’s not very good by today’s standards! Fox telem probably assumes the ideal 2.5V so each bit is 2.5/4096= 610uV. However, for example, SN 41 -Z MPPT Vref measured 2.486V which is really 2.486/4096=607uV per bit. This adds up over 4096 bits!

So in summary you can perform two single point calibrations for every panel of every mppt with the following information:
·       Vref measurement I took at room temperature

·       Vrtd with associated Vtemp voltage (presented at ADC input)

With this you can calibrate out the voltage reference initial tolerance error as well as calibrate out the gain error in the signal conditioning circuitry for each panels Y=-mX+b amplifier.

**Comment by ac2cz** [comment]
The temperature is not coming out correctly.  This can be applied once we work out what the issue is with the temp.  Down graded to High and removed from Release 1.05 as not critical to fly.

**Comment by ac2cz** [comment]
This is working for RadFxSat.  Need to make sure the right conversions are applied to 1D

**Comment by ac2cz** [comment]
In RadFxSat is see that I added two calibration values to the Spacecraft file:
mpptResistanceError=6.58
mpptSensorOffThreshold=1600

Those value do not seem to be in the Fox-1D spacecraft file.

**Comment by ac2cz** [comment]
In 1.06h
